packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -1589,7 +1589,7 @@ void main() {
       ),
     );
 
-    final RenderEditable render = tester.allRenderObjects
+    final RenderEditable render = tester.allRenderObjects.whereType<RenderEditable>()
```

Comment:
It could but it change a little the semantic. With `tester.allRenderObjects.whereType<RenderEditable>().first` you could get a child of `RenderEditable` and not exactly a `RenderEditable`. I've only kept the original semantic.

Reply:
It looks like for this test it wouldn't matter, so I would prefer the simplified version.

Reply:
Done